charts/substra-backend/templates/rbac.yaml

```diff
@@ -20,9 +20,12 @@ rules:
     resources: ["secrets"]
     verbs: ["get", "watch", "list", "create", "update", "patch"]
   - apiGroups: [""]
-    resources: ["pods", "pods/log", "pods/status", "pods/exec"]
+    resources: ["pods/log", "pods/status"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: [""]
+    resources: ["pods", "pods/exec"]
     verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
-  - apiGroups: ["batch", "extensions"]
+  - apiGroups: ["batch"]
```

Comment:
Do we actually use batch jobs in the worker?

Reply:
Ctrl+F-ing `batch` in the code base would lead my to believe we don't

Reply:
But do we need to run jobs in the worker? I thought we just ran pods :x

Reply:
To the best of my knowledge, we don't use K8s Jobs currently